code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -279,14 +389,15 @@
 					replaced_wound = existing_wound
 
 		if(initial(possible_wound.threshold_minimum) < injury_roll)
+			var/datum/wound/new_wound
 			if(replaced_wound)
-				var/datum/wound/new_wound = replaced_wound.replace_wound(possible_wound)
+				new_wound = replaced_wound.replace_wound(possible_wound)
 				log_wound(owner, new_wound, damage, wound_bonus, bare_wound_bonus, base_roll)
 			else
-				var/datum/wound/new_wound = new possible_wound
+				new_wound = new possible_wound
 				new_wound.apply_wound(src)
 				log_wound(owner, new_wound, damage, wound_bonus, bare_wound_bonus, base_roll)
```

Comment:
If apply wound fails and deletes the wound, it will still attempt to log the null wound. You need to check for this, it is causing runtimes.

Reply:
log_wound is also causing runtimes in cases where owner is null, for instance if the bodypart has been dismembered.